drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -189,3 +189,6 @@ validationSucceed=Validation Succeed
 
 backgroundTabTitle=Background
 backgroundErrorNotification=There are errors in Background data, please have a look
+ruleCheatSheet23=To specify a MVEL expression (or invoke a Java method) just put # at the beginning of the expression. In GIVEN section, return type of the expression has to be the same one of type of the column. In EXPECT section the type could also just a boolean with the result of the assertion and you can use actualValue identifier to access value to check.
+ruleCheatSheetBackground=If the same GIVEN data is shared by multiple test scenarios, you can use the Background tab to define them once. You can create a column in Background in the same way of Model
+dmnCheatSheetBackground=If the same GIVEN data is shared by multiple test scenarios, you can use the Background tab to define them once. You can create a column in Background in the same way of Model.
```

Comment:
@danielezonca I agree with @dupliaka, I suggest to define a separate method (eg. `setCommonCheatSheetContent()`) where shared descriptions are set in one single point and with a single property definition, in order to avoid text duplication.

Reply:
I have removed the duplication (`commonCheatSheetBackground`) but I prefer not to refactor the whole method to create a setCommonCheatSheetContent because the structure of the HTML is different. To do that we should refactor the whole panel

Reply:
Also since we discussed it with you @danielezonca  the dmn and drl bkg are not the same; 
So drl can use it as a batch update but dmn can not and dmn would take only last record of it